site/app/templates/admin/admin_gradeable/AdminGradeableDates.twig

```diff
@@ -1,3 +1,6 @@
+<div class="warning-banner content" id="gradeable-dates-warnings" data-testid="gradeable-dates-warnings-banner">
+    <p class="dates-warning" id="grade-inquiry-dates-warning" data-testid="grade-inquiry-dates-warning">Warning: Grade Inquiry ends before it starts. Students will not be able to make Grade Inquires.</p>
+</div>
```

Comment:
Will this briefly show for a few ms while loading? Or could it? Might be better to start it off as hidden.